docs/source/changelog.rst

```diff
@@ -18,6 +18,7 @@ Changelog
         * Fix issue with missing instance ids and categorical entity index (:pr:`1050`)
         * Remove warnings.simplefilter in feature_set_calculator to un-silence warnings (:pr:`1053`)
         * Fix feature visualization for features with '>' or '<' in name (:pr:`1055`)
+        * Fix ``EntitySet.plot()`` when given a dask entityset (:pr:`1086`)
```

Comment:
This line needs to be moved to the future releases section and you should add yourself as a contributor for next release